.claude/skills/release/SKILL.md

```
@@ -246,14 +312,32 @@ gh release create vx.y.z --title "vx.y.z" --notes "{changelog section content wi
+git fetch origin
+git merge-base --is-ancestor origin/main origin/dev \
+  && echo "dev contains main — OK" \
+  || { echo "STILL DIVERGED"; git log origin/dev..origin/main --oneline; }
 ```
 
+> **`--no-rebase` is required, not optional.** Without it, git aborts with
+> `fatal: Need to specify how to reconcile divergent branches` on any machine
+> that has not set `pull.rebase`. It also pins the behaviour to the merge this
+> step actually wants — a `pull.rebase=true` config would otherwise rewrite dev's
+> history, which is exactly what the warning below forbids.
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)